mantidproject/mantid
# Move plotting and saving out of indirect algorithms

Fixes #13000.

To test:
- Code review
- Try a few of the interfaces with combinations of plotting and saving enabled, files should be saved into the default save directory as whatever the output workspace name is.

How to run some of the interfaces can be found on several other PRs:
- #767
- #12920
- #12892
- #12837

No release notes updates as this is an internal change.
